logs-to-tsv/README.md

```diff
@@ -0,0 +1,15 @@
+# Cloudwatch Logs to TSV
+This folder holds two script files. query-logs.js queries AWS Cloudwatch for approximately 200,000 GET requests that look like initial searches from the Discovery Front End. It then writes the api requests to logs-to-tsv/logs-out. write-logs-tsv.js takes the files in that directory and writes them to a tsv.
+
+## How to use
+`cd logs-to-tsv`
+`nvm use`
```

Comment:
Seems like you also need a local `package.json` with `"@aws-sdk/client-cloudwatch-logs"` as a `devDependency` so that - within the `logs-to-tsv` directory - one can call `nvm use; npm i` to install script-specific deps